docs/userGuide/syntax/variables.mbdf

```diff
@@ -58,19 +58,9 @@ Represents the root directory of the project. Used for specifying intra-site lin
 
 <code>{<span></span>{ timestamp }}</code> is the time stamp that indicates when the page was generated.
 
-You can set the time zone and locale in `site.json` like so - 
-
-```js
-site.json
-{
-  ...
-  "timeZone": "Asia/Singapore",
-  "locale": "en-US",
-  ...
-}
-```
+Compiled in the following gists are lists of possible [time zone](https://gist.github.com/yash-chowdhary/136df3f6721f2fd1d5ea46b14f7c531c) and [locale](https://gist.github.com/yash-chowdhary/674654bdd977b063de008c1b3655a515) options.
```

Comment:
Hmm why not add it into the docs itself?
Perhaps a minimal panel or link leading to a new page in the docs

Reply:
Added it in a minimal panel 👍